tools/cli/admin.go

```diff
@@ -1002,6 +1003,11 @@ func getDBFlags() []cli.Flag {
 			Name:  FlagTLSEnableHostVerification,
 			Usage: "cassandra tls verify hostname and server cert (tls must be enabled)",
 		},
+		cli.GenericFlag{
+			Name:  FlagConnectionAttributes,
+			Usage: "a key-value set of sql database connection attributes (must be in key=value format)",
```

Comment:
what is the format for multiple KVs? Can you add an example of one KV and two KVs?

Reply:
Yes, it should be `a=b,c=d,...,x=y` or just `a=b`. I've updated the Usage.